### ISSUE TITLE
Where's smalldict?

### ISSUE THREAD

**Issue by mulka** [stanographer/steno-dictionaries#1]
In the docs it references smalldict. Wondering where that is.

**Comment by stanographer** [comment]
Smalldict is just a json file with {}.

It's a template for a new dictionary.

**Comment by stanographer** [comment]
Further instructions can be found at: https://github.com/openstenoproject/plover/wiki/Dictionary-Format

**Comment by mulka** [comment]
Got it. I thought it might be something more exciting like a bare-bones dictionary with just the main letters and sounds.

**Comment by stanographer** [comment]
Yeah, I don't know if there's a dictionary like that. I think it's just best to have the full repertoire at your disposal and just use the lessons as a guide to go step-by-step. But to create a new dictionary, all you have to do is touch a new file and contents should just be {}. Save, point Plover to it, and new entries should save without a hitch!

**Comment by kolkatasouth001** [comment]
Hello Stanley, I am an eager lerner of steno from India with lack of resources. I have 2 specific questions...1) suppose I want to use plover or dotterel (mobile version of steno app since I don't have the actual machine to practice)for practicing some foreign language steno like french or spanish, can I use this like that? If yes then what will be the process for that? How to install foreign dictionaries into plover or dotterel? Can you tell me the process. 2)can You give me any link from where I can download this dictionaries...I am asking you this because I have seen your videos in youtube stenoing in different langusges other than english...can you help me with this. Thanks.
